castervoice/lib/ccr/dragonfly_dev_commands/dragonfly_dev_commands.py

```diff
@@ -0,0 +1,191 @@
+            + Function(type_mimic),
+                rdescript="automatically create mimic command with given spec"),
+        "command [<spec>] playback [<text>]": R(Text('"%(spec)s": ,') + Key("left") 
+            + Function(type_playback), 
+                rdescript="automatically create playback command with given spec"),
+            # the command above has been inconsistent. maybe because it's 
+            # automatically putting in two of each parable character e.g. brackets
+            # might have to adjust it depending on the editor
+        
+        "command [<spec>] mouse [<mouse_button>]": R(Text('"%(spec)s": ,') + Key("left")
+            + Function(type_mouse_current_position_button, extra={"mouse_button"}), 
+                rdescript="automatically create a command to click at the current mouse position with given spec"),
+            # for some reason the above command is not putting in the left click by default. perhaps someone can fix this
+        
+        # same as above but uses the caster standard format with the R and rdescript.
+            # maybe somebody knows how to make it so that you can tab through the relevant places
+        "commander [<spec>] key": R(Text('"%(spec)s": R(Key(""), rdescript=""')  + Key("right, comma") + Key("left:18"),
+            rdescript="caster: automatically create key command with given spec"),
+        "commander [<spec>] text": R(Text('"%(spec)s": R(Text(""), rdescript=""')  + Key("right, comma") + Key("left:18"),
+            rdescript="caster: automatically create text command with given spec"),
+        "commander [<spec>] [bring] app": R(Text('"%(spec)s": R(BringApp(r), rdescript=""')  + Key("right, comma") + Key("left:17"),
+            rdescript="caster: automatically create bing app command with given spec"),
+        "commander function [<spec>]": R(Text('"%(spec)s": R(Function(), rdescript=""')  + Key("right, comma") + Key("left:17"),
+            rdescript="caster: automatically create function command with given spec"),
+        "commander [<spec>] mimic [<text>]": R(Text('"%(spec)s": R(')
+            + Function(type_mimic) + Text(', rdescript=""')  + Key("right, comma, left:3"),
+            rdescript="caster: automatically create mimic command with given spec"),
+        "commander [<spec>] mouse [<mouse_button>]": R(Text('"%(spec)s": R(') 
+            + Function(type_mouse_current_position_button, extra={"mouse_button"})
+                + Key("right") + Text(', rdescript=""') + Key("right, comma, left:3"),
+            rdescript="caster: automatically create command to click at current mouse position with given spec"),
+        
+        # I couldn't get the one for playback to work
+        # "commander [<spec>] playback [<text>]": Text('"%(spec)s": R(') # 
+         #   + Function(type_playback) + Key("down:2") + Text(", rdescript=''") + Key("right, comma, left:3"),        
+        
+
+
+
+        # Miscellaneous.
```

Comment:
These are actually already implemented in CCR punctuation.py

        "[<long>] equals":
            R(Text("%(long)s" + "=" + "%(long)s"), rdescript="Equals Sign"), 
        "[<long>] plus":
            R(Text("%(long)s" + "+" + "%(long)s"), rdescript="Plus Sign"),
        "[<long>] minus":
            R(Text("%(long)s" + "-" + "%(long)s"), rdescript="Dash"),

Edit punctuation.py to could implement the long "%(long)s for pipe 
        "pipe (sim | symbol)":
            R(Text("|"), rdescript="Pipe Symbol"),
Deckle could be modified in a similar way.
        "deckle":
            R(Text(":"), rdescript="Colon"),

Doing this keeps redundancy down and keeps cognitive load low.